docs/java-rest/high-level/indices/update_aliases.asciidoc

```diff
@@ -1,10 +1,10 @@
 [[java-rest-high-update-aliases]]
```

Comment:
Should the section shortcut also change to "indices-aliases"?

Reply:
no please :) it does not matter that much, and both names are correct :)

Reply:
one more inconsistency not mentioned above: the way we reference our API in our docs may differ from the request names, the client method names and the api names in our SPEC :)  here we try to name pages the same as how they are named in the es reference